### ISSUE TITLE
"valueLink" issue for "LongTextInput"

### ISSUE THREAD

**Issue by TarSzator** [marmelab/admin-on-rest#1135]
**What you were expecting:**
I did not want to receive the following console warning

> Warning: React does not recognize the `valueLink` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `valuelink` instead. If you  accidentally passed it from a parent component, remove it from the DOM element.

**What happened instead:**
I got the console warning ;-)

**Steps to reproduce:**
It is probably enough to use the "LongTextInput" component but I got it with my own component (see code section)

**Related code:**
```
import _ from 'lodash'
import React, {Component} from 'react'
import PropTypes from 'prop-types'
import {connect} from 'react-redux'
import {Tab, Tabs} from 'material-ui/Tabs'
import * as aor from 'admin-on-rest'
import {Field} from 'redux-form';

let {LongTextInput, showNotification} = aor

class LanguageObjectInput extends Component {
	render() {
		const props = this.props
		let {record, source, resource} = props
		let languageObject = record[source] || {}
		let tabs = _.map(languageObject, (value, key) => {
			let id = `${resource}-${source}-${key}`
			return <Tab label={key} key={`tab-${id}`}>
				<div>
					<Field
						key={`field-${id}`}
                                                name={`${source}.${key}`}
                                                component={LongTextInput}
                                                type="string"
						placeholder={`Translation for "${key}" for property "${source}"`}
					/>
				</div>
			</Tab>
		})
		return <Tabs>{tabs}</Tabs>;
	}
}

LanguageObjectInput.propTypes = {
	record: PropTypes.object,
	source: PropTypes.string.isRequired
}

export default connect(null, {})(LanguageObjectInput)
```

**Other information:**
It might has something todo with this issue and could might be fixed with an react dependancy update
https://github.com/callemall/material-ui/issues/7779

**Environment**

* Admin-on-rest version: 1.3.2
* React version: 16.0.0
* Browser: Chrome 61.0.3163.100 (Official Build) (64-bit)
* Stack trace (in case of a JS error):
Warning: React does not recognize the `valueLink` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `valuelink` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
    in textarea (created by EnhancedTextarea)
    in div (created by EnhancedTextarea)
    in EnhancedTextarea (created by TextField)
    in div (created by TextField)
    in TextField (created by LongTextInput)
    in LongTextInput (created by ConnectedField)
    in ConnectedField (created by Connect(ConnectedField))
    in Connect(ConnectedField) (created by Field)
    in Field (at language-object-input.js:34)
    in div (at language-object-input.js:33)
    in div (created by TabTemplate)
    in TabTemplate (created by Tabs)
    in div (created by Tabs)
    in div (created by Tabs)
    in Tabs (at language-object-input.js:41)
    in LanguageObjectInput (created by Connect(LanguageObjectInput))
    in Connect(LanguageObjectInput) (at dynamic-input.js:31)
    in DynamicInput (created by Connect(DynamicInput))
    in Connect(DynamicInput) (at index.js:33)
    in FormFieldComponent (created by FormField)
    in FormField (created by FormInput)
    in div (created by FormInput)
    in FormInput (created by SimpleForm)
    in div (created by SimpleForm)
    in form (created by SimpleForm)
    in SimpleForm (created by Form(SimpleForm))
    in Form(SimpleForm) (created by Connect(Form(SimpleForm)))
    in Connect(Form(SimpleForm)) (created by ReduxForm)
    in ReduxForm (created by Connect(ReduxForm))
    in Connect(ReduxForm) (at index.js:31)
    in div (created by Card)
    in div (created by Paper)
    in Paper (created by Card)
    in Card (created by Edit)
    in div (created by Edit)
    in Edit (created by WithPermissionsFilteredChildren)
    in WithPermissionsFilteredChildren (created by getContext(WithPermissionsFilteredChildren))
    in getContext(WithPermissionsFilteredChildren) (created by getContext(getContext(WithPermissionsFilteredChildren)))
    in getContext(getContext(WithPermissionsFilteredChildren)) (created by Connect(getContext(getContext(WithPermissionsFilteredChildren))))
    in Connect(getContext(getContext(WithPermissionsFilteredChildren))) (at index.js:30)
    in ConfigEdit (created by Route)
    in Restricted (created by Connect(Restricted))
    in Connect(Restricted) (created by Route)
    in Route (created by CrudRoute)
    in Switch (created by CrudRoute)
    in CrudRoute (created by Route)
    in Route (created by AdminRoutes)
    in Switch (created by AdminRoutes)
    in AdminRoutes (created by Connect(AdminRoutes))
    in Connect(AdminRoutes) (created by getContext(Connect(AdminRoutes)))
    in getContext(Connect(AdminRoutes)) (created by Layout)
    in div (created by Layout)
    in div (created by Layout)
    in div (created by Layout)
    in div (created by Layout)
    in MuiThemeProvider (created by Layout)
    in Layout (created by WithWidth)
    in EventListener (created by WithWidth)
    in WithWidth (created by Connect(WithWidth))
    in Connect(WithWidth) (created by Route)
    in Route (created by Admin)
    in Switch (created by Admin)
    in div (created by Admin)
    in Router (created by ConnectedRouter)
    in ConnectedRouter (created by Admin)
    in TranslationProvider (created by withContext(TranslationProvider))
    in withContext(TranslationProvider) (created by Connect(withContext(TranslationProvider)))
    in Connect(withContext(TranslationProvider)) (created by Admin)
    in Provider (created by Admin)
    in Admin (created by withContext(Admin))
    in withContext(Admin) (at App.js:51)
    in App (at index.js:16)

**Comment by Inoeng85** [comment]
any updated on this issue?..im facing same issue with exactly same environtment..

**Comment by NikitaVlaznev** [comment]
Same env, same problem.
Using just LongTextInput.

**Comment by djhi** [comment]
Linked to #1139

**Comment by djhi** [comment]
As we have updated to `material-ui` `~0.19.0` (note the `~`) and this has been fixed in `material-ui` `0.19.4`, it should be fixed for us too. Feel free to reopen if needed
